### PR TITLE
Update Set-CsTeamsVoiceApplicationsPolicy.md

### PR DESCRIPTION
**Pull request by CLYVR**
Updates for new policy keys - also re-org'd to group into AA/CQ/Metrics

### CODE REVIEW

**Comment by CLYVR** [comment]
@DaniEASmith Please and thank you!

**Comment by Stacyrch140** [comment]
Hi @CLYVR  - 

**Following is a blocking issue that needs to be resolved before we can merge:**

teams/teams-ps/teams/New-CsTeamsVoiceApplicationsPolicy.md
teams/teams-ps/teams/Set-CsTeamsVoiceApplicationsPolicy.md
- These articles don't have Acrolinx scorecards. 

***

**Reference**

[Criteria used for pull request reviews](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=main)

**Blocking**
- All articles in the PR have an Acrolinx score of 80 or higher. Limited exceptions are available. For more information, see [Minimum Acrolinx topic scores for publishing](https://review.learn.microsoft.com/en-us/office-authoring-guide/acrolinx-min-score).

***

After you resolve the blocking issues, remove the **Back to submitter** label and add the **Sign off** label again. 

If you need help resolving issues in this PR, post your question in the [Ask an Admin Teams Channel](https://teams.microsoft.com/l/channel/19%3aea9a561f603c4fbb9cf3931a6849eae4%40thread.skype/Ask%2520an%2520Admin?groupId=7079ffbf-1420-4ee2-9ddc-a2c351337320&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47).


Thanks, 
Stacy

**Comment by DaniEASmith** [comment]
> Hi @CLYVR -
> 
> **Following is a blocking issue that needs to be resolved before we can merge:**
> 
> teams/teams-ps/teams/New-CsTeamsVoiceApplicationsPolicy.md teams/teams-ps/teams/Set-CsTeamsVoiceApplicationsPolicy.md
> 
> * These articles don't have Acrolinx scorecards.
> 
> **Reference**
> 
> [Criteria used for pull request reviews](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=main)
> 
> **Blocking**
> 
> * All articles in the PR have an Acrolinx score of 80 or higher. Limited exceptions are available. For more information, see [Minimum Acrolinx topic scores for publishing](https://review.learn.microsoft.com/en-us/office-authoring-guide/acrolinx-min-score).
> 
> After you resolve the blocking issues, remove the **Back to submitter** label and add the **Sign off** label again.
> 
> If you need help resolving issues in this PR, post your question in the [Ask an Admin Teams Channel](https://teams.microsoft.com/l/channel/19%3aea9a561f603c4fbb9cf3931a6849eae4%40thread.skype/Ask%2520an%2520Admin?groupId=7079ffbf-1420-4ee2-9ddc-a2c351337320&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47).
> 
> Thanks, Stacy

How do we get Acrolinx scorecard to show up? This wasn't something that either of us removed. I don't know how that would occur.

**Comment by tiburd** [comment]
This is a public repo PR; as of now, it's not expected to have an Acrolinx scorecard.

I manage a team of Upwork freelance partners who will review this PR; after it's approved, I'll merge it.

File: teams/teams-ps/teams/New-CsTeamsVoiceApplicationsPolicy.md [hunk 1]
```diff
@@ -59,15 +67,15 @@ TeamsVoiceApplicationsPolicy is used for Supervisor Delegated Administration whi
 New-CsTeamsVoiceApplicationsPolicy -Identity SDA-Allow-CQ-Moh -AllowCallQueueMusicOnHoldChange $true
```
[the remark this answers is not in the document]
Comment (CLYVR) [reply teams/teams-ps/teams/New-CsTeamsVoiceApplicationsPolicy.md:67]: @tseward The examples are meant to be directly executed.  Yes the identity would be unique to the cx's environment however identity is shown in other policies.  Happy to change if needed but just following what was already there / what is done elsewhere.

### -------------------------- Example 1 --------------------------
```
Set-CsOnlineVoicemailPolicy -Identity "CustomOnlineVoicemailPolicy" -MaximumRecordingLength ([TimeSpan]::FromSeconds(60))
```

### Example 1
```powershell
PS C:\> Set-CsTeamsShiftsPolicy -Identity OffShiftAccess_WarningMessage1_AlwaysShowMessage -ShiftNoticeMessageType Message1 -ShiftNoticeFrequency always -AccessGracePeriodMinutes 5
```

And in cmdlets:

### -------------------------- Example 1 -------------------------- 
```
Set-CsCallQueue -Identity e7e00636-47da-449c-a36b-1b3d6ee04440 -UseDefaultMusicOnHold $true
```

File: teams/teams-ps/teams/New-CsTeamsVoiceApplicationsPolicy.md [hunk 2]
```diff
@@ -247,9 +255,10 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -AllowCallQueueMusicOnHoldChange
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsVoiceApplicationsPolicy.md:258]:
```suggestion
```

File: teams/teams-ps/teams/New-CsTeamsVoiceApplicationsPolicy.md [hunk 3]
```diff
@@ -471,21 +498,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+
+
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsVoiceApplicationsPolicy.md:502]:
```suggestion
```

File: teams/teams-ps/teams/New-CsTeamsVoiceApplicationsPolicy.md [hunk 4]
```diff
@@ -519,6 +548,74 @@ Accept pipeline input: False
...
 
+### -RealTimeAutoAttendantMetricsPermission
+
+PARAMVALUE: None | Authorized | All
+
+When set to `None` (the default value), users affected by the policy won't receive real-time metrics for auto attendants.
+
+When set to `Authorized`, users affected by the policy will receive real-time metrics for auto attendants they are authorized for.
+ 
+When set to `All`, users affected by the policy will receive real-time metrics for all auto attendants in the tenant.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsVoiceApplicationsPolicy.md:559]:
```suggestion
When set to `All`, users affected by the policy will receive real-time metrics for all auto attendants in the organization.
```

File: teams/teams-ps/teams/New-CsTeamsVoiceApplicationsPolicy.md [hunk 5]
```diff
@@ -519,6 +548,74 @@ Accept pipeline input: False
...
+ 
+### -RealTimeCallQueueMetricsPermission
+
+PARAMVALUE: None | Authorized | All
+
+When set to `None` (the default value), users affected by the policy won't receive real-time metrics for call queues.
+
+When set to `Authorized`, users affected by the policy will receive real-time metrics for call queues they are authorized for.
+ 
+When set to `All`, users affected by the policy will receive real-time metrics for all call queues in the tenant.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsVoiceApplicationsPolicy.md:582]:
```suggestion
When set to `All`, users affected by the policy will receive real-time metrics for all call queues in the organization.
```

File: teams/teams-ps/teams/New-CsTeamsVoiceApplicationsPolicy.md [hunk 6]
```diff
@@ -519,6 +548,74 @@ Accept pipeline input: False
...
+
+### -RealTimeAgentMetricsPermission
+
+PARAMVALUE: None | Authorized | All
+
+When set to `None` (the default value), users affected by the policy won't receive real-time metrics for agents.
+
+When set to `Authorized`, users affected by the policy will receive real-time metrics for agents who are members in the call queues they are authorized for.
+ 
+When set to `All`, users affected by the policy will receive real-time metrics for all agents in all queues call queues in the tenant.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsVoiceApplicationsPolicy.md:605]:
```suggestion
When set to `All`, users affected by the policy will receive real-time metrics for all agents in all call queues in the organization.
```

File: teams/teams-ps/teams/Set-CsTeamsVoiceApplicationsPolicy.md [hunk 7]
```diff
@@ -14,24 +14,48 @@ Modifies an existing Teams voice applications policy.
...
+ [-RealTimeAgentMetricsPermission <string>]
+ 
+ [[-Identity] <String>]
  [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
 
-TeamsVoiceApplicationsPolicy is used for Supervisor Delegated Administration which allows tenant admins to permit certain users to make changes to auto attendant and call queue configurations.
+`TeamsVoiceApplicationsPolicy` is used for **Supervisor Delegated Administration** which allows tenant admins to permit certain users to make changes to auto attendant and call queue configurations.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsVoiceApplicationsPolicy.md:58]:
```suggestion
`TeamsVoiceApplicationsPolicy` is used for **Supervisor Delegated Administration** which allows admins in the organization to permit certain users to make changes to auto attendant and call queue configurations.
```

File: teams/teams-ps/teams/Set-CsTeamsVoiceApplicationsPolicy.md [hunk 8]
```diff
@@ -429,50 +471,119 @@ Accept pipeline input: False
...
+### -RealTimeAutoAttendantMetricsPermission
+
+PARAMVALUE: None | Authorized | All
 
-When set to True users affected by the policy will be allowed to change the call queue's conference mode. When set to False (the default value) users affected by the policy will not be allowed to change the call queue's conference mode.
+When set to `None` (the default value), users affected by the policy won't receive real-time metrics for auto attendants.
+
+When set to `Authorized`, users affected by the policy will receive real-time metrics for auto attendants they are authorized for.
+ 
+When set to `All`, users affected by the policy will receive real-time metrics for all auto attendants in the tenant.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsVoiceApplicationsPolicy.md:530]:
```suggestion
When set to `All`, users affected by the policy will receive real-time metrics for all auto attendants in the organization.
```

File: teams/teams-ps/teams/Set-CsTeamsVoiceApplicationsPolicy.md [hunk 9]
```diff
@@ -429,50 +471,119 @@ Accept pipeline input: False
...
+ 
+### -RealTimeCallQueueMetricsPermission
+
+PARAMVALUE: None | Authorized | All
+
+When set to `None` (the default value), users affected by the policy won't receive real-time metrics for call queues.
+
+When set to `Authorized`, users affected by the policy will receive real-time metrics for call queues they are authorized for.
+ 
+When set to `All`, users affected by the policy will receive real-time metrics for all call queues in the tenant.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsVoiceApplicationsPolicy.md:553]:
```suggestion
When set to `All`, users affected by the policy will receive real-time metrics for all call queues in the organization.
```

File: teams/teams-ps/teams/Set-CsTeamsVoiceApplicationsPolicy.md [hunk 10]
```diff
@@ -429,50 +471,119 @@ Accept pipeline input: False
...
+
+### -RealTimeAgentMetricsPermission
+
+PARAMVALUE: None | Authorized | All
+
+When set to `None` (the default value), users affected by the policy won't receive real-time metrics for agents.
+
+When set to `Authorized`, users affected by the policy will receive real-time metrics for agents who are members in the call queues they are authorized for.
+ 
+When set to `All`, users affected by the policy will receive real-time metrics for all agents in all queues call queues in the tenant.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsVoiceApplicationsPolicy.md:576]:
```suggestion
When set to `All`, users affected by the policy will receive real-time metrics for all agents in all call queues in the organization.
```

File: teams/teams-ps/teams/New-CsTeamsVoiceApplicationsPolicy.md [hunk 11]
```diff
@@ -10,46 +10,54 @@ ROBOTS: NOINDEX
...
+ 
  [-WhatIf] 
  [-Confirm]
  [<CommonParameters>]
 ```
 
 ## DESCRIPTION
 
-TeamsVoiceApplicationsPolicy is used for Supervisor Delegated Administration which allows tenant admins to permit certain users to make changes to auto attendant and call queue configurations.
+`TeamsVoiceApplicationsPolicy` is used for **Supervisor Delegated Administration** which allows tenant admins to permit certain users to make changes to auto attendant and call queue configurations.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsVoiceApplicationsPolicy.md:60]:
```suggestion
`TeamsVoiceApplicationsPolicy` is used for **Supervisor Delegated Administration** which allows admins in the organization to permit certain users to make changes to auto attendant and call queue configurations.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @CLYVR, kindly commit the suggested minor changes. Many thanks.

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Pull request merged**